app/controllers/api/invitations_controller.rb

```diff
@@ -5,7 +5,11 @@ def create
                                                      group: @group,
                                                      inviter: current_user,
                                                      message: invitation_form_params[:message])
-    respond_with_collection
+    if @invitations.empty? and email_addresses.any?
```

Comment:
If it fires whether there are email addresses or not then we'd need to update the error message - what should it be? @robguthrie, what do you think?